examples/high_level_functions_basic_objects.js

```diff
@@ -0,0 +1,253 @@
+*********************************************************************************************/
+
+run("../includes/Tools/clearAll.js");
+
+var material = createMaterial("S235");
+var section = createSection(material, "IPE 80");
+var section2 = createSection(material, "IPE 100");
+var nodesForMembers = createNodesGrid(-28, -28, [10, 6], [3, 4]);
+if (RFEM) {
+	var nodeForLines = createNodesGrid(-28, -6, [10, 10], [3, 4]);
+	var thickness = createThickness("0.250", material, thicknesses.TYPE_UNIFORM);
+	var solid = makeSolid([[8, -16, 0], [20, -16, 0], [20, -8, 0], [8, -8, 0], [8, -16, -5], [20, -16, -5], [20, -8, -5], [8, -8, -5]]);
+	var nodesForSurfaces = createNodesGrid(1, 10, [10, 10], [3, 2]);
+	var linesForSurfaces = createSurfacesFromNodesGrid(nodesForSurfaces, [5, 5], undefined, undefined, true);
+
+	 /*For line welded joints
+					line1			line2
+				+-------------+-------------+node3
+			   /node1		 /|	node2	   /
+			  /				/ |			  /
+			 /			   /  |			 /line3
+	  line6	/		line7 /	  |	line9   /
+		   /			 /	  |		   /
+		  /	node6		/node5|		  /
+		 +----line5----+-----line4---+node4
+					   |	  |
+							  +node8
+					   |	  /
+				line8  |	 /
+					   |	/line10
+					   |   /
+					   |  /
+					   | /
+					   +node7
+	*/
+	var node1 = createNode(8, -5, 0);
+	var node2 = createNode(13, -5, 0);
+	var node3 = createNode(18, -5, 0);
+	var node4 = createNode(18, 0, 0);
+	var node5 = createNode(13, 0, 0);
+	var node6 = createNode(8, 0, 0);
+	var node7 = createNode(node5.coordinate_1, node5.coordinate_2, 5);
+	var node8 = createNode(node2.coordinate_1, node2.coordinate_2, 5);
+	var line1 = createLine(node1.no, node2.no);
+	var line2 = createLine(node2.no, node3.no);
+	var line3 = createLine(node3.no, node4.no);
+	var line4 = createLine(node5.no, node4.no);
+	var line5 = createLine(node6.no, node5.no);
+	var line6 = createLine(node6.no, node1.no);
+	var lineForWeldedJoint = new Line();
+	var lineForWeldedJointNo = lineForWeldedJoint.Polyline(undefined, [node5.no, node2.no]).no;
+	var line8 = createLine(node5.no, node7.no);
+	var line9 = createLine(node2.no, node8.no);
+	var line10 = createLine(node7.no, node8.no);
+	var surfaceForWeldedJoint = createSurface([line1.no, lineForWeldedJointNo, line5.no, line6.no], surfaces.TYPE_STANDARD, thickness);
+	var surface2ForWeldedJoint = createSurface([line2.no, line3.no, line4.no, lineForWeldedJointNo], surfaces.TYPE_STANDARD, thickness);
+	var surface3ForWeldedJoint = createSurface([lineForWeldedJointNo, line9.no, line10.no, line8.no], surfaces.TYPE_STANDARD, thickness);
+}
+
+/*************************************** Members *********************************************/
+var member = new Member();
+if (RFEM) {
+	// Member defined by line
+	var line = new Line(undefined, [1, 2]);
+	member.Beam(undefined, line.no, 1);
+} else {
+	// Member defined by nodes
+	member.Beam(undefined, [1, 2], 2);
+}
+member.Rigid(undefined, [3, 4], "Beam member");
```

Comment:
I prefer eg. new Member().Beam(....) but it is nitpick

Reply:
Toto teď nevím, jak to udělat. Asi by každý typ prutu musel mít vlastní konstruktor... Můžeme se o tom pobavit.